### PR TITLE
Add internal function for WayScript terminal output

### PR DESCRIPTION
**Pull request by jdhayford**
## Checklist

- [x] Application changes have been tested and automated tests have been added, if applicable
- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM python:3.10 AS base
 LABEL org.wayscript.image.authors="founders@wayscript.com"
 
-RUN apt-get update && apt-get -yu dist-upgrade
+RUN apt-get update && apt-get -yu dist-upgrade freetds-dev
```
Comment (aroxby-wayscript) [inline Dockerfile:4]: Is this change related to the new function?

**Review by aroxby-wayscript — APPROVED** [review]
LGTM

**Comment by jeffbuswell** [comment]
> LGTM
> 
> I'm a little concerned about the endpoint. Is it new? It looks like anybody can add output to any process?

We should probably check that the auth token matches the service id/process if we're no longer doing m2m auth on that endpoint

**Comment by jeffbuswell** [comment]
Why add this to the python library, instead of the terminal application?

File: Dockerfile (hunk 1 above)
Comment (jdhayford) [reply Dockerfile:4]: This change was made to fix a currently existing build error that has occurred likely due to remote dependency changes
